apis/r/R/TileDBGroup.R

```diff
@@ -98,7 +106,8 @@ TileDBGroup <- R6::R6Class(
         }
 
         spdl::debug("Closing {} '{}'", self$class(), self$uri)
-        tiledb::tiledb_group_close(private$.tiledb_group)
+        ##tiledb::tiledb_group_close(private$.tiledb_group)
+        c_group_close(private$.tiledb_group)
```

Comment:
Can we clean out the commented code?
```suggestion
        c_group_close(private$.tiledb_group)
```